test-site/config/global_config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "sdkVersion": "1.8", // The version of the Answers SDK to use
+  "sdkVersion": "develop", // The version of the Answers SDK to use
```

Comment:
just checking this is intentional!

Reply:
yes, we need to be on develop for this to work